regression-test/suites/external_table_p0/hive/ddl/test_hive_ddl_and_ctas.groovy

```diff
@@ -217,75 +216,372 @@ suite("test_hive_ddl_and_ctas", "p0,external,hive,external_docker,external_docke
+                            "location_uri" = "/user/hive/warehouse/test_ctas_ex/loc_ctas_o2",
+                            "file_format"="orc",
+                            "orc.compress"="zlib"
+                        )
+                        AS SELECT null as col1, pt2 as col2, pt1 FROM internal.test_ctas_olap.part_ctas_olap_src WHERE col1>0;
+                    """
+                sql """ INSERT INTO ${catalog_name}.test_ctas_ex.ctas_o2 (col1,pt1,col2)
+                        SELECT col1,pt1,pt2 FROM internal.test_ctas_olap.part_ctas_olap_src;
+                """
+                sql """ INSERT INTO ${catalog_name}.test_ctas_ex.ctas_o2 (col2,pt1,col1)
+                        SELECT pt2,pt1,col1 FROM internal.test_ctas_olap.part_ctas_olap_src;
+                """
+                sql """ INSERT OVERWRITE TABLE ${catalog_name}.test_ctas_ex.ctas_o2 (pt1,col2)
+                        SELECT pt1,col1 FROM internal.test_ctas_olap.part_ctas_olap_src;
+                """
+                order_qt_ctas_04 """ SELECT * FROM ctas_o2  """
+                sql """ DROP TABLE ctas_o2 """
+            } finally {
+                destroySrcDDLForCTAS(catalog_name)
+            }
+        }
+
+        def test_ddl_exception = { String file_format, String catalog_name ->
+            sql """ switch ${catalog_name} """
+
+            sql """ create database if not exists `test_hive_ex` """;
+            try {
+                sql """ create database `test_hive_ex` """
+            } catch (Exception e) {
+                suite.getLogger().warn(e.getMessage())
+            }
+            sql """use `${catalog_name}`.`test_hive_ex`"""
+
+            sql """
+                CREATE TABLE test_hive_ex.ex_tbl_${file_format}(
+                  `col1` BOOLEAN COMMENT 'col1',
+                  `col2` INT COMMENT 'col2',
+                  `col3` BIGINT COMMENT 'col3',
+                  `col4` CHAR(10) COMMENT 'col4',
+                  `col5` FLOAT COMMENT 'col5',
+                  `col6` DOUBLE COMMENT 'col6',
+                  `col7` DECIMAL(6,4) COMMENT 'col7',
+                  `col8` VARCHAR(11) COMMENT 'col8',
+                  `col9` STRING COMMENT 'col9',
+                  `pt1` VARCHAR COMMENT 'pt1',
+                  `pt2` STRING COMMENT 'pt2',
+                  `pt3` DATE COMMENT 'pt3'
+                )  ENGINE=hive 
+                PARTITION BY LIST (pt1, pt2) ()
+                PROPERTIES (
+                  'file_format'='${file_format}'
+                )
+            """;
+
+            try {
+            // test  columns
+            sql """ INSERT INTO ex_tbl_${file_format} (`col1`, `col2`, `col3`, `col4`, `col5`, `col6`, `col7`, `col8`, `col9`) 
+                    VALUES 
+                    (true, 123, 987654321099, 'abcdefghij', 3.1214, 63.28, 123.4567, 'varcharval', 'stringval');
+                """
+            } catch (Exception e) {
+                suite.getLogger().warn(e.getMessage())
```

Comment:
You didn't check anything here.
It will pass no matter the exception throw or not.

Reply:
Use `test action`
See: https://doris.apache.org/community/developer-guide/regression-testing#test-action